desc/compute/utils.py

```diff
@@ -343,16 +343,25 @@ def get_transforms(keys, obj, grid, jitable=False, **kwargs):
         if hasattr(obj, c + "_basis"):  # regular stuff like R, Z, lambda etc.
             basis = getattr(obj, c + "_basis")
             # first check if we already have a transform with a compatible basis
-            for transform in transforms.values():
-                if basis.equiv(getattr(transform, "basis", None)):
-                    ders = np.unique(
-                        np.vstack([derivs[c], transform.derivatives]), axis=0
-                    ).astype(int)
-                    # don't build until we know all the derivs we need
-                    transform.change_derivatives(ders, build=False)
-                    c_transform = transform
-                    break
-            else:  # if we didn't exit the loop early
+            if not jitable:
+                for transform in transforms.values():
+                    if basis.equiv(getattr(transform, "basis", None)):
```

Comment:
What is this checking (if this is the check you were referencing below)? Is it important?

Reply:
R, Z and lambda, for example each need their own transform to compute the basis functions to calculate them. however usually these are all the same resolution (L, M, N) and Z and lambda are always the same symmetry (`"sin"` if stell symmetric) and in the case of a stellarator asymmetric equilibrium, R Z and lambda are all asymmetric bases. So, we dont need to have 2 (or 3) separate transforms for these quantiites, we can just reuse the same one (if stell sym, that means Z and lambda can share a transform, if asymmetric, means that all 3 can use the same transform)

the check here is checking if we can do that. It is nice to save some memory but is not completely necessary so I think it is fine to skip the check whenever `jitable=True` is passed